### ISSUE TITLE
Is image URL parsing broken?

### ISSUE THREAD

**Issue by Alphare** [Kozea/CairoSVG#166]
Or am I missing something?

I have a very simple document:
```svg
<?xml version="1.0" encoding="utf-8"?>
<svg height="500" id="svg3488" preserveAspectRatio="xMidYMin meet" style="position: relative;"
     version="1.1" viewBox="0 0 700 500" width="700" xmlns="http://www.w3.org/2000/svg"
     xmlns:xlink="http://www.w3.org/1999/xlink">
    <g id="layer_1">
        <rect height="100%" id="background" style="fill: white;" width="100%" x="0" y="0"/>
    </g>
    <g id="layer_2">
        <text fill="black" font-family="Arial" font-size="30" transform="translate(100, 100)">Votre
            texte ici
        </text>
        <image height="165"
               xlink:href="https://images-client-123imprim-dev.s3.amazonaws.com/savage/tmp/bff00e6a1dbb494b84b1ebf4f42b6ed8/recto"
               transform="translate(237, 108)" width="250"/>
    </g>
</svg>
```
When I call `cairosvg.svg2pdf(bytestring=bytes(svg, encoding="utf-8"), write_to="/tmp/test_save.pdf")` (with `svg` being the above document as a string), and set a breakpoint in `cairosvg/image.py` line 38, I see that `node.get('{http://www.w3.org/1999/xlink}href')` is `None`. Shouldn't it be  `node.get('href')`? Why is it failing if not?

When I change to simply `node.get('href')`, the URL is resolved correctly.

Thanks. :)

**Comment by liZe** [comment]
Your code is working for me.

`{http://www.w3.org/1999/xlink}href` is the [Clark notation](http://www.jclark.com/xml/xmlns.htm) of the `xlink:href` when the `xlink` namespace is defined as `http://www.w3.org/1999/xlink`. Everything is OK with your file, your code and CairoSVG's code. I don't know why you don't get your image.

Does it work with the `cairosvg` executable?

**Comment by Alphare** [comment]
Ah, I figured that string was a notation of some sort, thanks.
It appears to work with the executable, to my surprise! 

Looking more into the issue, I've found that the svg was not always the same (shame on me). The difference was the namespace declaration `xmlns:xlink="http://www.w3.org/1999/xlink"` and its corresponding effect on the `href` (`xlink:href`). 

I suppose this is expected behavior and this issue should be closed. :)
Thank you for the quick response!
